docs/index.md

```diff
@@ -10,7 +10,7 @@ page_keywords: docker, machine, amazonec2, azure, digitalocean, google, openstac
 > **Note**: Machine is currently in beta, so things are likely to change. We
 > don't recommend you use it in production yet.
 
-Machine makes it really easy to create Docker hosts on your computer, on cloud
+Machine makes it really simple to create Docker hosts on your computer, on cloud
```

Comment:
I changed this because @fredlf once warned me to not say things like, "Hey, look how easy that was!" to users in docs.  Since if they end up having trouble, they feel stupid / angry.